lib/ansible/modules/cloud/azure/azure_rm_managed_disk.py

```diff
@@ -41,50 +41,45 @@
         choices:
             - absent
             - present
-        required: false
     location:
         description:
             - Valid Azure location. Defaults to location of the resource group.
         default: resource_group location
-        required: false
     storage_account_type:
         description:
             - "Type of storage for the managed disk: 'Standard_LRS'  or 'Premium_LRS'. If not specified the disk is created 'Standard_LRS'"
         choices:
             - Standard_LRS
             - Premium_LRS
-        required: false
     create_option:
         description:
             - "Allowed values: empty, import, copy. 'import' from a VHD file in 'source_uri' and 'copy' from previous managed disk 'source_resource_uri'."
         choices:
             - empty
             - import
             - copy
-        required: false
     source_uri:
         description:
             - URI to a valid VHD file to be used when 'create_option' is 'import'.
-        required: false
     source_resource_uri:
         description:
             - The resource ID of the managed disk to copy when 'create_option' is 'copy'.
-        required: false
     os_type:
         description:
             - "Type of Operating System: 'linux' or 'windows'. Used when 'create_option' is either 'copy' or 'import' and the source is an OS disk."
         choices:
             - linux
             - windows
-        required: false
     disk_size_gb:
         description:
-            -Size in GB of the managed disk to be created. If 'create_option' is 'copy' then the value must be greater than or equal to the source's size.
-        required: true
+            - Size in GB of the managed disk to be created. If 'create_option' is 'copy' then the value must be greater than or equal to the source's size.
```

Comment:
A few things when writing descriptions

* Put each new sentence on a new line
* When referencing another module option, enclose it in `I()`
* When referencing a value for an option, enclose it in `C()`
* You don't need it here but you can reference another module by enclosing it in `M()`
* You don't need it here but you can add in a URL by enclosing it in `U()`

So in the end this would look like

```
description:
- Size in GB of the managed disk to be created.
- If I(create_option) is C(copy), then the value must be greater than or equal to the source's size.
```